edb/edgeql-rust/src/normalize.rs

```diff
@@ -145,11 +145,17 @@ pub fn normalize<'x>(text: &'x str)
                 push_var(&mut rewritten_tokens, "__std__::float64",
                     next_var(variables.len()),
                     tok.start, tok.end);
+                let value = tok.value.replace("_", "").parse()
+                    .map_err(|e| Error::Tokenizer(
+                        format!("can't parse std::float64: {}", e),
+                        tok.start))?;
+                if value == f64::INFINITY || value == -f64::INFINITY {
+                    return Err(Error::Tokenizer(
+                        format!("number is out of range for std::float64"),
```

Comment:
What error would be raised for `SELECT <std::float32>1e10000`?  Would it be "number is out of range for std::float64" or "number is out of range for std::float32"? The latter would be less confusing to users.

Reply:
We don't have float32 syntax. it's basically just `<float32>` cast value with float64 syntax. On float32 postgres does the job:
```
edgedb> SELECT <float32>1e99;
ERROR: NumericOutOfRangeError: value out of range: overflow
edgedb> SELECT 1e99;
{1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000}
```

On the other hand, we can use bare `float` to make it less confusing.

Reply:
> ERROR: NumericOutOfRangeError: value out of range: overflow

This is a nice enough error, all is good